app/assets/javascripts/discourse/app/components/modal/spreadsheet-editor.gjs

```diff
@@ -190,15 +210,19 @@ export default class SpreadsheetEditor extends Component {
               heading.length * rowWidthFactor,
               this.defaultColWidth
             ),
-            align: "left",
           };
         });
       } else {
+        if (this.alignments == null) {
+          this.alignments = this.extractTableAlignments(row);
+        }
         // rows:
         rows.push(this.extractTableContent(row));
       }
     });
 
+    headings.forEach((h, i) => (h.align = this.alignments?.[i] ?? "left"));
```

Comment:
I wonder, how do tables work in RTL languages? 🤔 Should the default be `right` in those cases?

No need to "fix" this in this PR, more like a "food for thought" question.

Reply:
Yes, this is what I am also curious about. But in the previous implementation, it is always `left`

```javascript
headings = this.extractTableContent(row).map((heading) => {
          return {
            title: heading,
            width: Math.max(
              heading.length * rowWidthFactor,
              this.defaultColWidth
            ),
            align: "left",
          };
        });
```